uber/custom_tags.py

```diff
@@ -16,7 +16,7 @@ def timestamp(dt):
 
 @register.filter
 def jsonize(x):
-    return SafeString(json.dumps(x, cls=serializer))
+    return SafeString(html.escape(json.dumps(x, cls=serializer), quote=False))
```

Comment:
I expect that the reason we initially USED SafeString was because, when we escape quotes in JSON, it breaks a whoooooole lot of fun stuff. Luckily, the html.escape method lets you bypass that.